data-processor/src/main/java/io/micronaut/data/processor/mappers/jta/JtaTransactionalMapper.java

```diff
@@ -37,7 +37,7 @@ public class JtaTransactionalMapper implements NamedAnnotationMapper {
     @NonNull
     @Override
     public String getName() {
-        return "javax.transaction.Transactional";
+        return "jakarta.transaction.Transactional";
```

Comment:
This should be reverted, it converts javax TX. We should probably have a test that javax transactional can be still used

Reply:
Hm, yes, didn't see there was already jakarta. Reverted this change and added test here https://github.com/micronaut-projects/micronaut-data/pull/2265/commits/da71b0faf6ac6df681adfe3509a188dbb2963153. Hope it makes sense